Bio/Align/_aligners.c

```diff
@@ -0,0 +1,6049 @@
+/* Copyright 2018 by Michiel de Hoon.  All rights reserved.
+ * This code is part of the Biopython distribution and governed by its
+ * license.  Please see the LICENSE file that should have been included
+ * as part of this package.
```

Comment:
Can you dual license this new file please?

Reply:
Done